py/sigver/verify.py

```diff
@@ -1,85 +1,88 @@
 # SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
 # SPDX-License-Identifier: Apache-2.0
+""" Signature Verification Script """
+import argparse
+
+import logging
 
 import sys
+import os
 
-from azure.identity import DefaultAzureCredential
-from azure.keyvault.certificates import CertificateClient
-from azure.keyvault.keys import KeyClient
-from azure.keyvault.keys.crypto import CryptographyClient, SignatureAlgorithm
+import json
+import base64
+import requests
 
 from sha256tree import sha256sum
 
-KEY_VAULT_URL = "https://ghaf-devenv-ca.vault.azure.net/"
-CERTIFICATE_NAME = "."
+LOG = logging.getLogger(os.path.abspath(__file__))
+
 
+# Default certificate to be used if no argument is given
+CERTIFICATE_NAME="INT-Ghaf-Devenv-Common"
+
+# Azure Function (verify-signature) URL
+URL = "https://ghaf-devenv-microsign-aleksandrtserepo-app.azurewebsites.net/api/verify-signature"
 
 def show_help():
+    """Show help messsage"""
```

Comment:
Note: this function `show_help(...)` is no longer needed after you moved to using `argparse`.